packages/query-core/src/queryCache.ts

```diff
@@ -117,7 +117,7 @@ export class QueryCache extends Subscribable<QueryCacheListener> {
 
     if (!query) {
       query = new Query({
-        cache: this,
+        client,
```

Comment:
@juliusmarminge will this break `trpc` ? Are you using `new Query` directly somewhere? If you’re doing `queryCache.build()`, nothing is changing ...

Reply:
Dont think so